coreneuron/mechanism/eion.cpp

```diff
@@ -189,6 +189,10 @@ the USEION statement of any model using this ion\n",
 #define FARADAY     _faraday_codata2018
 #define gasconstant _gasconstant_codata2018
 #endif
+
+// extern variables require acc declare
+#pragma acc declare create(celsius)
```

Comment:
I'm wondering if this should be moved in `register_mech.cpp` and do the same for `secondorder` as well? https://github.com/BlueBrain/CoreNeuron/blob/14ad572b5df4a67df419840261e4aa3893dee54b/coreneuron/mechanism/register_mech.cpp#L25  Or there is some issue with moving that there?

Reply:
Replied offline:
* at the definition site, every variable needs to have acc declare (copyin or create depending on initial value)
* at any other compilation unit, if variable is extern then it should also have acc declare
https://docs.nvidia.com/hpc-sdk/pgi-compilers/18.10/x86/openacc-gs/index.htm#openacc-globals